tests/monotouch-test/Security/KeyTest.cs

```diff
@@ -119,8 +119,13 @@ public void RoundtripRSAMinPKCS1 ()
 
 						using (var pub = public_key.GetPublicKey ())
 						{
+#if !MONOMAC
 							// a new native instance of the key is returned (so having a new managed SecKey is fine)
 							Assert.That (pub.Handle == public_key.Handle, Is.Not.EqualTo (v), "public/GetPublicKey");
+#else
+							// a new native instance of the key is returned (so having a new managed SecKey is fine)
+							Assert.That(pub.Handle == public_key.Handle, Is.Not.EqualTo(TestRuntime.CheckMacSystemVersion(10, 13) ? false : v), "public/GetPublicKey");
+#endif
```

Comment:
that's rather confusing. `v` is already equals to `TestRuntime.CheckMacSystemVersion (10, 13)` so it looks like the condition is `if (true ? false : true)` for 10.13
and `if (false : false : false)` or 10.12

let's simplify the condition and kill the lambda, they do not help the readability of the code